lib/services/types.proto

```diff
@@ -750,6 +750,14 @@ message RoleConditions {
 
     // KubeUsers is an optional kubernetes users to impersonate
     repeated string KubeUsers = 7 [ (gogoproto.jsontag) = "kubernetes_users,omitempty" ];
+
+    // ClusterLabels is a map of node labels (used to dynamically grant access to
+    // clusters).
+    wrappers.LabelValues ClusterLabels = 8 [
+        (gogoproto.nullable) = false,
```

Comment:
shouldn't this field be nullable?

Reply:
because we are using custom type, I think it will generate *LabelValues